Task_TheGridProblem/index.css

```diff
@@ -1 +1,33 @@
-/* Write code below for GRID PROBLEM TASK */
+<html>
+    <head>
+        <title>The Grid Problem</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+        <style>
+        .dash-container{
+          display:grid;
+          grid-template-columns:auto auto auto;
+          grid-template-rows:300px 200px 100px 200px;
+           width:800px;
+          }
+         .dash-item{
+          background-color:#E7C4A2;
+          text-align:center;
+          
+          }
+        </style>
+    </head>
+    <body>
+     <div class="dash-container">
+    <div class="dash-item">one</div>

```

Comment:
Consider moving inline styles to an external CSS file for better separation of concerns and maintainability.